charts/thoras/templates/api-server-v2/deployment.yaml

```diff
@@ -38,8 +38,6 @@ spec:
         name: thoras-v2-api-server
         command: ["/app/api-server", "-p", {{ .Values.thorasApiServerV2.containerPort | quote }}]
         env:
-          - name: SERVICE_REASONING_API_URL
```

Comment:
I moved this so the env field we're testing is in the same array index across all deployments . This let's us to have one helm unit test instead of many and the env var ordering shouldn't matter